### PR TITLE
add tasks, validators, errors

### PR DESCRIPTION
**Pull request by AlmazYarullin**

### CODE REVIEW

File: app/controllers/tasks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    @task = Task.new
+  end
+
+  def edit
+  end
+
+  def update
+    if @task.update(task_params)
+      redirect_to task_path(@task), notice: 'Task has been updated'
+    end
```
Comment (DmitryBarskov) [inline app/controllers/tasks_controller.rb]: What happens if task is failed to update?

File: app/views/tasks/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  <%= link_to @task.project.name, url_for(@task.project)%>
+</p>
+
+<p>
+  <strong>Deadline:</strong>
+  <%= @task.deadline_at %>
+</p>
+
+<%= link_to 'Edit', edit_task_path(@task) %>
+<%= link_to 'Back', tasks_path %>
```
Comment (DmitryBarskov) [inline app/views/tasks/show.html.erb]: It would be convenient to see destroy action here.

**Review by DmitryBarskov — CHANGES_REQUESTED** [review]

File: app/controllers/tasks_controller.rb (hunk 1 above)
Comment (AlmazYarullin) [reply app/controllers/tasks_controller.rb]: Ooops, it should render edit page. I'll fix it

File: app/views/tasks/show.html.erb (hunk 2 above)
Comment (AlmazYarullin) [reply app/views/tasks/show.html.erb]: It will be done

**Review by DmitryBarskov — APPROVED** [review]

**Pull request closed without merging**